platforms/core-runtime/native/src/main/java/org/gradle/internal/nativeintegration/services/NativeServices.java

```diff
@@ -226,6 +226,48 @@ private void initializeNativeIntegrations(File userHomeDir, NativeIntegrationEna
         }
     }
 
+    private static boolean shouldUseNativeIntegration(NativeIntegrationEnabled nativeIntegrationEnabledCondition) {
+        if (!nativeIntegrationEnabledCondition.isEnabled()) {
+            return false;
+        }
+        if (isLinuxWithMusl()) {
+            LOGGER.info("Linux with musl libc detected, disabling native services");
```

Comment:
🤔  Should this be debug?

Reply:
Good point, I made it look more like the other messages signaling that we are not compatible with `native-platform`.